WooCommerce/Classes/ViewModels/Order Details/OrderDetailsViewModel.swift

```diff
@@ -681,24 +691,24 @@ extension OrderDetailsViewModel {
     }
 
     @MainActor
-    func syncShippingLabels() async {
+    func syncShippingLabels() async -> [ShippingLabel]? {
```

Comment:
This method can be marked as `private` as we don't access this from outside of this view model.

Reply:
Marking as resolved as we access this method from unit tests.